ci-operator/config/openshift/kubernetes-sigs-lws/openshift-kubernetes-sigs-lws-main.yaml

```diff
@@ -41,6 +41,19 @@ tests:
   skip_if_only_changed: ^\.tekton/|\.md$|^(LICENSE|OWNERS)$
   steps:
     workflow: openshift-ci-security
+- as: test-e2e-aws
+  skip_if_only_changed: ^\.tekton/|\.md$|^(LICENSE|OWNERS)$|^site/
+  steps:
+    cluster_profile: aws
```

Comment:
Would you want to specify what OCP version you are testing?

it isn't clear what cluster you are getting?

Reply:
OCP version is defined in https://github.com/openshift/release/blob/f19e7da89be58ad317795bf01ca437f612ad4f8c/ci-operator/config/openshift/kubernetes-sigs-lws/openshift-kubernetes-sigs-lws-main.yaml#L6-L14

Reply:
Yes but when you get a aws cluster do you know what version of OCP it is?

For kueue we specify version. I'm not sure what the default is here.

Reply:
Good question. As far as I know, it is the latest accepted ci or nightly.